dao/rhc_connection_dao_test.go

```diff
@@ -322,3 +322,44 @@ func TestRhcConnectionListForSources(t *testing.T) {
 
 	DoneWithFixtures(RHC_CONNECTION_SCHEMA)
 }
+
+// TestRhcConnectionRowsClosed is a regression test for https://issues.redhat.com/browse/RHCLOUD-18192. It tests that
+// when there are no rows to process from a result set, a proper response is returned instead of a "rows are closed"
+// error.
+func TestRhcConnectionRowsClosed(t *testing.T) {
+	testutils.SkipIfNotRunningIntegrationTests(t)
+	CreateFixtures(RHC_CONNECTION_SCHEMA)
+
+	dbRhcConnections := make([]model.RhcConnection, 0)
+	err := DB.Debug().
+		Model(&model.RhcConnection{}).
+		Find(&dbRhcConnections).
+		Error
+
+	if err != nil {
+		t.Errorf(`want nil error, gokt "%s"`, err)
```

Comment:
gokt -> got

Reply:
Nice catch, thank you! :smile_cat: :+1:

Reply:
actually it looks that it is word from Dutch :)